include/chemfiles/formats/mmCIF.hpp

```diff
@@ -48,9 +48,6 @@ class mmCIFFormat final: public Format {
     std::map<std::string, size_t> atom_site_map_;
     /// Map of residues, indexed by residue id and chainid.
     std::map<std::pair<std::string, int64_t>, Residue> residues_;
-    /// Set to true if the file is based on fractional coordinates.
-    /// Set to false if the file is based on cartesian coordinates.
-    bool uses_fract_;
```

Comment:
Why is this removed? I think in theory mmCIF can use fractional coordinates.

Reply:
No, mmCIF only defines `Cartn_x` and friends: https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v40.dic/Categories/atom_site.html

CIF files define both `Cartn_x` and `fract_x`: https://www.iucr.org/__data/iucr/cifdic_html/1/cif_core.dic/Iatom_site_Cartn_.html / https://www.iucr.org/__data/iucr/cifdic_html/1/cif_core.dic/Iatom_site_fract_.html, but these files are handled by the GEMMI based CIF reader.